src/upnp.cpp

```diff
@@ -109,7 +109,11 @@ namespace upnp {
       IGDdatas data;
       urls_t urls;
       std::array<char, INET6_ADDRESS_STRLEN> lan_addr;
+#if (MINIUPNPC_API_VERSION >= 18)
```

Comment:
Can we create a function that returns the status, and does this if inside? I really don't like the duplication.